.editorconfig

```diff
@@ -10,3 +10,7 @@ indent_size = 4
 
 [{package.json,*.yml}]
 indent_size = 2
+
+[*.toml]
+indent_size = 2
+max_line_length = 79
```

Comment:
Because 79 print characters + LF?
I dunno; I saw it was supposed to 79 not 80 someplace for something, so I did it here too.